arb-gateway/wrangler.toml

```diff
@@ -1,4 +1,4 @@
-name = "arb-gateway-worker"
+name = "arb-sepolia-gateway-worker"
```

Comment:
Should this be changed?

Reply:
yes. I changed to add testnet info so that `arb-gateway` is used for mainnet

Reply:
Right, but this is not a sepolia-specific file; what will we do when deploying to mainnet?

Reply:
Turned out that I can add environment https://developers.cloudflare.com/workers/wrangler/configuration/#environmental-variables so changed accordingly